firebase-config/firebase-config.gradle

```diff
@@ -58,6 +58,9 @@ dependencies {
     implementation 'com.google.firebase:firebase-measurement-connector:18.0.0'
     implementation 'com.google.android.gms:play-services-tasks:17.0.2'
 
+    implementation 'com.squareup.okhttp3:okhttp:4.9.0'
```

Comment:
This version is written in kotlin so you might want to downgrade to something like 3.14

Reply:
I was actually thinking given the overall size increase from the library (+127%) that we don't try to use HTTP/2 and just use 1.1 so that we can just use the JDK provided HttpUrlConnection class to make a connection. Then we would just use chunked encoding to simulate streaming from server to client. The PR with this idea is here: https://github.com/firebase/firebase-android-sdk/pull/3451